src/simularium/VisData.ts

```diff
@@ -376,15 +381,29 @@ class VisData {
         }
     }
 
+    public setCachePreferences(disableCache: boolean): void {
+        this.disableCache = disableCache;
+    }
+
```

Comment:
See above note about renaming `disableCache`! I think here I wouldn't know what `setCachePreferences(true)` or `setCachePreferences(false)` mean if I saw it elsewhere in the codebase. You could either change this to `setCacheEnabled()`, or accept an object (`{ cacheEnabled: boolean }`) to make this more clear.

Reply:
Ok I can change this! I was anticipating that in future this function might also take in max sizes, but I definitely see what you mean here